README.md

```diff
@@ -0,0 +1,29 @@
+# GO REST API
```

Comment:
Great job having a README with instructions on how to run. You could also add steps that show how to download dependencies